libs/wire-subsystems/src/Wire/UserSubsystem.hs

```diff
@@ -96,3 +116,11 @@ getUserProfile luid targetUser =
 getLocalUserProfile :: (Member UserSubsystem r) => Local UserId -> Sem r (Maybe UserProfile)
 getLocalUserProfile targetUser =
   listToMaybe <$> getLocalUserProfiles ((: []) <$> targetUser)
+
+getLocalUserAccount :: (Member UserSubsystem r) => Local UserId -> Sem r (Maybe UserAccount)
+getLocalUserAccount uid =
+  listToMaybe
+    <$> getAccountsBy
+      ( qualifyAs uid $
+          def {getByUserIds = [tUnqualified uid]}
+      )
```

Comment:
this looks very similar to the thing I said above - maybe this should be the impl of that effect op? what am I missing?